app/models/replication_transfer.rb

```diff
@@ -99,6 +100,25 @@ def cancel!(reason)
 
   private
 
+  # True if we created this request
+  def we_requested
+    from_node && from_node.namespace == Rails.configuration.local_namespace
+  end
```

Comment:
Consider the idiom that boolean methods end with `?`.  Possible name alternatives for this method might be `we_requested?` or `our_transfer?` or `my_transfer?` or `we_initiated?` or `we_own_transfer?` etc.